README.md

```diff
@@ -94,29 +85,8 @@ const password = `@keychain:AC_PASSWORD`;
 
 You can obtain an API key from [Appstore Connect](https://appstoreconnect.apple.com/access/api). Create a key with _App Manager_ access. Note down the Issuer ID and download the `.p8` file. This file is your API key and comes with the name of `AuthKey_<appleApiKeyId>.p8`. This is the string you have to supply when calling `notarize`.
 
-Based on the `ApiKey`, the legacy `altool` will look in the following places for that file:
-
-* `./private_keys`
-* `~/private_keys`
-* `~/.private_keys`
-* `~/.appstoreconnect/private_keys`
-
 `notarytool` will not look for the key, and you must instead provide its path as the `appleApiKey` argument.
```

Comment:
since notarytool is the only option, we can replace this sentence with "You must provide its path as the `appleApiKey` argument."